CHANGES.md

```diff
@@ -27,6 +27,9 @@
       constraint (was: `IO`), due to changes in how the xmonad core handles XDG
       directories.
 
+    - The prompt window now sets a `WM_CLASS` property.  This allows to
+      match on it in a manageHook.
```

Comment:
Seems to me you also forgot what the motivation was. xmonad wouldn't be stupid enough to try to manage its own windows would it? This was more about shadows and rounded corners in a compositor, I think.

Reply:
Oh :)  Very good point yes, I'll change that